web/src/main/ui/app/components/flows-new/edit-flow/mapping/ui/mapping-ui.component.html

```diff
@@ -1,7 +1,9 @@
 <!-- START NEW MAPPING VIEW -->
 <div *ngIf="isVersionCompatibleWithES" class="source-prop-container-map-exp">
-
-  <div>
+  <div *ngIf="targetEntity && !targetEntity.hasDocs">
+    <em>Unable to find documents that match your query.</em>
+  </div>
```

Comment:
In the old mapping  UI, if no docs are found, the info we see is "Before configuring this mapping you must ingest source documents." . Can we continue to have that or is there any reason for the change ?

Reply:
I checked the message suggested in the user story and it matched the implementation here.